### DISCUSSION TITLE
Running plocate service on WSL-2 Nix

### DISCUSSION THREAD

**Discussion by mwoodpatrick** [nix-community/NixOS-WSL#540] [Q&A]
Does anyone know how to run the plocate service on WSL-2 Nix. I read a number of threads (listed below) but was still unclear on how to achieve this

https://github.com/NixOS/nixpkgs/pull/156185

https://github.com/NixOS/nixpkgs/pull/124260

https://discourse.nixos.org/t/plocate-unusable/34257

**Comment by SuperSandro2000** [comment]
I used something like in the past:

```nix
services.locate = {
  enable = true;
  localuser = null; # silence warning
  package = pkgs.plocate; # use faster locate implementation
  prunePaths = [
    "/media"
    "/mnt/c" # don't index windows drives in WSL
    "/mnt/d"
    "/mnt/e"
    "/mnt/f"
    "/mnt/wsl"
    "/nix/store"
    "/nix/var/log/nix"
    "/tmp"
    "/var/spool"
    "/var/tmp"
  ];
};
```
